pkg/utils/keypath/key_path.go

```diff
@@ -321,7 +321,7 @@ func svcRootPath(svcName string) string {
 	return path.Join(constant.MicroserviceRootPath, c, svcName)
 }
 
-// LegacyRootPath returns the root path of legacy pd service.
+// LegacyRootPath returns the root path of legacy PD.
```

Comment:
Do we need to replace legacy with non-microservice env?

Reply:
LegacyRootPath uses "legacy", I just keep using it.